packages/orm/src/EntityManager.ts

```diff
@@ -1086,8 +1088,15 @@ export class EntityManager<C = unknown, Entity extends EntityW = EntityW> {
       const baseMeta = getBaseMeta(getMetadata(entity));
       const { createdAt, updatedAt } = baseMeta.timestampFields;
       const { data } = getInstanceData(entity);
-      if (createdAt) data[createdAt] = new Date();
-      if (updatedAt) data[updatedAt] = new Date();
+      const now = Temporal.Now.instant();
```

Comment:
Huh, I was thinking it'd be nice to stay with `new Date()` to avoid the dependency, but I guess this gets us nano precision, which would be nice for the `lastNow` handling...

If the `instant()` was still millis precision, I'd be tempted to still use `new Date()` to avoid the runtime dependency.